addons/crm/static/src/views/crm_form/crm_form.js

```diff
@@ -80,7 +79,7 @@ class CrmFormController extends formView.Controller {
 
     async onRecordSaved(record) {
         if (this.changedStage) {
-            await checkRainbowmanMessage(this.orm, this.effect, record.resId);
+            await checkRainbowmanMessage(this.model.orm, this.effect, record.resId);
```

Comment:
In a component, we will avoid using a service that comes from another object. In your case, we will avoid using the orm service from the model. For two reasons:
- It is custom so it could have some particular behavior
- It is not encapsulated by _protectMethod (in useService). This method will bind the life cycle of your component to the service in order to stop the calls to the service if the component is destroyed. 

So you should always use useService in a component.